Assignment-22/countryStateDistrict/districtTriggerHelper

```diff
@@ -0,0 +1,34 @@
+public with sharing class districtTriggerHelper {
+    /*
+    ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+    * @Description      This method is used when an district is inserted deleted or updated it updates total no of district on state object.
+    * @author           Mohd Owais  <mohd.owais@fexle.com>
+    * @Return           void
+    * @Parameter        List,Map
+    * @created          2022-08-13
+    * ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+    */
+    public static void insertUpdateDeleteDistrict(List<District__c> newDistrict,Map<Id,District__c> oldDistrictMap){
```

Comment:
Is this method inserting, Updating or deleting district records? The name should be like rollUpDistrictCountOnState